services/dynamic-sidecar/requirements/_test.in

```diff
@@ -6,6 +6,7 @@ pytest
 pytest-cov
 pytest-asyncio
 pytest-mock
+pytest-lazy-fixture
```

Comment:
I think we agreed we should not use this library right? :-) Didn't we had some serious troubles ... perhaps i am confused. @sanderegg ??

Reply:
OPS, removed it.